setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name="rhumba",
-    version='0.1.18',
+    version='0.2.0',
```

Comment:
No I fucking hate letters in my versions. It causes all kinds of issues with things that fail to parse it